datasets/naturalearth.yaml

```diff
@@ -0,0 +1,28 @@
+Name: Natural Earth
+Description: Natural Earth is a public domain map dataset available at 1:10m, 1:50m, and 1:110 million scales. Featuring tightly integrated vector and raster data, with Natural Earth you can make a variety of visually pleasing, well-crafted maps with cartography or GIS software.
+Documentation: https://www.naturalearthdata.com/downloads/
+Contact: https://github.com/nvkelso/natural-earth-vector/issues
+ManagedBy: North American Cartographic Information Society (nacis.org)
+UpdateFrequency: As needed
+Tags:
+  - aws-pds
+  - mapping
+  - geospatial
+  - global
+  - tiles
+  - earth observation
+  - population
+License: Public Domain. https://www.naturalearthdata.com/about/terms-of-use/
+Resources:
+  - Description: Shapefile downloads in ZIP archives
+    ARN: arn:aws:s3:::naturalearth
+    Region: us-west-2
+    Type: S3 Bucket
+    RequesterPays: False
+    ControlledAccess: False
```

Comment:
You can just delete this one, it expects a string here for information on how to access the data if its controlled access

Reply:
Fixed via https://github.com/awslabs/open-data-registry/pull/853/commits/2a81509b8fb068253a75dcf9afc4d888048e831a